btcjson/chainsvrcmds_test.go

```diff
@@ -530,6 +530,33 @@ func TestChainSvrCmds(t *testing.T) {
 			},
 		},
 		{
+			name: "gettxoutproof",
+			newCmd: func() (interface{}, error) {
+				return btcjson.NewCmd("gettxoutproof", []string{"123", "456"})
+			},
+			staticCmd: func() interface{} {
+				return btcjson.NewGetTxOutProofCmd([]string{"123", "456"}, nil)
+			},
+			marshalled: `{"jsonrpc":"1.0","method":"gettxoutproof","params":[["123","456"]],"id":1}`,
+			unmarshalled: &btcjson.GetTxOutProofCmd{
+				TxIDs: []string{"123", "456"},
+			},
+		},
+		{
+			name: "gettxoutproof optional",
+			newCmd: func() (interface{}, error) {
+				return btcjson.NewCmd("gettxoutproof", []string{"123", "456"}, btcjson.String("block"))
```

Comment:
Not a big deal, but I'd probably use something more representative of a hash instead of "block".